float32 alpha
float32 x
float32 y
float32 z
bool tag_located
float32 marker_displacement
int32 room_marker_tracked
